src/types/purchaseOrder.ts

```diff
@@ -0,0 +1,10 @@
+interface PurchaseOrder {
```

Comment:
please use kebab case when naming your files. Instead of naming it "purchaseOrder.ts", it should be named "purchase-order.ts"

Reply:
done